etc/env/l1-inits/.init.env

```diff
@@ -1,29 +1,29 @@
 CONTRACTS_LATEST_PROTOCOL_VERSION=22
 CONTRACTS_CREATE2_FACTORY_ADDR=0x3c4CE2E9D8b03BbCd568E47465A483EE86893c49
-CONTRACTS_VERIFIER_ADDR=0xCf42AEFfB17dED8de2d4116D33716354F53e51e6
-CONTRACTS_L1_MULTICALL3_ADDR=0x69D85c64032337E58064A35020d6F45Dab350883
+CONTRACTS_VERIFIER_ADDR=0xF025a57cfda291db967Cbd88120fb8997232bC5E
```

Comment:
Not sure if this file was intended to be committed to the repo in the first place, looks like it should've been added to `.gitignore`.

Reply:
I reverted the changes to the file here [a19be60](https://github.com/matter-labs/zksync-era/pull/1360/commits/a19be607ea8b287339c040d78bc13d2ba8c8f18d), I believe it was commited in the shared bridge PR for some reason.